src/item.cpp

```diff
@@ -1238,7 +1238,7 @@ std::string Item::getDescription(const ItemType& it, int32_t lookDistance,
 		uint32_t volume = 0;
 		if (!item || !item->hasAttribute(ITEM_ATTRIBUTE_UNIQUEID)) {
 			if (it.isContainer()) {
-				volume = it.maxItems;
+				volume = item->getName() == "Quiver" ? 4 : it.maxItems;
```

Comment:
I think Item can be nullptr.

Reply:
This working fine. Just need to be attempted to slot system don't broke.

Reply:
This code will crash the server when you look at the item in npc trade.

Reply:
True.... I already start a new "new class" for "quivers"

Reply:
Okay, already test this script and are very obsolete, to come useful to version TFS 1.3 + Ammo type Verify, and some another things are completely wrong.
If anyone going to use that keep in mind this is a VERY RAW SCRIPT for you guys have some knowledge base, this is not usefully script C++.
This is complete unstable and can be very dangerous as these people don't have knowledge about what they are doing on the source...